stream-compositions/cursors/files/sql/transaction_cursor/mysql/create/txn_cursor.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `transaction_cursor`.`txn_cursor` (
+  `id` VARCHAR(36) NOT NULL,
+  `arrangement_id` VARCHAR(36) NOT NULL,
+  `ext_arrangement_id` VARCHAR(50) NOT NULL,
+  `last_txn_date` DATETIME NULL,
+  `last_txn_ids` VARCHAR(2600) NULL,
```

Comment:
I am sure there will be a good usecase for having these IDs here.

Reply:
They are to prevent an upsert call to the DBS API since that API does "INsert => If exception, rollback => Update". That's one too many operations which could be avoided by checking the list of last successfully ingested txns.